sheets/src/main/java/com/novoda/github/reports/sheets/network/SheetsServiceClient.java

```diff
@@ -0,0 +1,40 @@
+package com.novoda.github.reports.sheets.network;
+
+import com.novoda.github.reports.sheets.sheet.Entry;
+import com.novoda.github.reports.sheets.sheet.Sheet;
+
+import retrofit2.Response;
+import rx.Observable;
+import rx.functions.Func1;
+
+public class SheetsServiceClient {
+
+    // TODO extract to .json and read as prop
+    private static final String DOCUMENT_ID = "1rMeGnlugO312to0loBwN3x0QTvAxoHwv4Pe_SYXR1YE";
```

Comment:
yes, and the end one will also have to have public visibility ("only those with the link")